compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -4671,8 +4671,7 @@ abstract class GenJSCode extends plugins.PluginComponent
             tryGenRepeatedParamAsJSArray(arg, handleNil = false).fold {
               genExpr(arg)
             } { genArgs =>
-              genNew(WrappedArrayClass, WrappedArray_ctor,
-                  List(js.JSArrayConstr(genArgs)))
+              genWrappedArrayForVarargs(js.JSArrayConstr(genArgs))
```

Comment:
We now wrap vararg parameters into an `ImmutableArray` instead of a `WrappedArray`, because it has to be a `scala.collection.immutable.Seq` in 2.13.